package-lock.json

```diff
@@ -25150,9 +25150,9 @@
       }
     },
     "node_modules/zod": {
-      "version": "3.21.4",
-      "resolved": "https://registry.npmjs.org/zod/-/zod-3.21.4.tgz",
-      "integrity": "sha512-m46AKbrzKVzOzs/DZgVnG5H55N1sv1M8qZU3A8RIKbs3mrACDNeIOeilDymVb2HdmP8uwshOCF4uJ8uM9rCqJw==",
+      "version": "3.22.4",
```

Comment:
Revert change in this file

Reply:
@rishav-jha-mech It is not working after reverting the files
I also tried to create a new branch and commit changes in that branch but it is also giving same error message

Reply:
@MahendraDani in any case if we are not using zod, there is no point in upgrading its version in `package-lock.json`, you are saying its not working after changing the files, what do you mean by that ?, also have you tried deleting the `node_modules` folder and reinstalling packages ?

Reply:
@rishav-jha-mech I first unistalled `zod` using `npm uninstall --legacy-peer-deps zod` and then made the changes in `LoginPage.tsx` file.